src/pretix/static/pretixpresale/js/ui/main.js

```diff
@@ -383,29 +389,21 @@ $(function () {
     })
     $("select[id^=id_name_parts], input[id^=id_name_parts_], #id_email, #id_street, #id_company, #id_zipcode," +
         " #id_city, #id_country, #id_state").change(function () {
-        if (copy_to_first_ticket) {
-            var $first_ticket_form = $(".questions-form").first().find("[data-addonidx=0]");
-            $first_ticket_form.find("[id$=" + this.id.substring(3) + "]").val(this.value);
-            if (this.placeholder) {
-                $first_ticket_form.find("[placeholder='" + CSS.escape(this.placeholder) + "']").val(this.value);
-            }
-            var label = $("label[for=" + this.id +"]").first().contents().filter(function () {
-                return this.nodeType != Node.ELEMENT_NODE || !this.classList.contains("sr-only");
-            }).text().trim();
-            if (label) {
-                // match to placeholder and label
-                $first_ticket_form.find("[placeholder='" + CSS.escape(label) + "']").val(this.value);
-                var v = this.value;
-                $first_ticket_form.find("label").each(function() {
-                    var text = $(this).first().contents().filter(function () {
-                        return this.nodeType != Node.ELEMENT_NODE || !this.classList.contains("sr-only");
-                    }).text().trim();
-                    if (text == label) {
-                        $("#" + this.getAttribute("for")).val(v);
-                    }
-                });
-            }
-        }
+        if (!copy_to_first_ticket) return;
+
+        var source = this;
+        var source_label = source.placeholder || get_label_text_for_id(source);
```

Comment:
This removes an important? matching bit: with this change we either match to the placeholder of the source or to the label whereas originally we matched source.placeholder==input.placeholder (line 390) as well as source.label== input.placeholder (line 397). In most cases this does not make a difference as the placeholder is either the same as the label (most fields) or the placeholder is anyways the one we want to match (name_parts). There is one exception: „Address“ has a placeholder of „Street and Number“ and we want to match that input not only with our own system question (which also has „Address“ as label and „Street and Number“ as placeholder) but with existing custom questions which should also be matched when they are being labelled „Address“. With this change, they would need to be labelled „Street and Number“. I think this should be:

```suggestion
        var source_label = get_label_text_for_id(source);
```

Also see suggestion further down.